build.sbt

```diff
@@ -38,4 +38,7 @@ val root = project.in(file("."))
 scalacOptions := Seq("-unchecked", "-deprecation", "-feature")
 
 (scalastyleConfig in Test) := baseDirectory.value / "scalastyle-test-config.xml"
-scalastyleSources in Test ++= {(unmanagedSourceDirectories in Integration).value}
\ No newline at end of file
+scalastyleSources in Test ++= {(unmanagedSourceDirectories in Integration).value}
+
+fork := true
+javaOptions += "-Dkey-file-format=%s\n%s"
```

Comment:
We keep all our configuration in `application.conf` Why did we decide to read this particular setting from `javaOptions`?